ts/src/pro/kucoin.ts

```diff
@@ -579,6 +600,20 @@ export default class kucoin extends kucoinRest {
         market = this.safeMarket (marketId, market);
         const symbol = market['symbol'];
         const side = this.safeStringLower (order, 'side');
+        // find previous trades in order
+        const stored = this.orders;
+        const ordersById = this.safeValue (stored, symbol, {});
+        const storedOrder = this.safeValue (ordersById, id);
+        const trades = this.safeValue (storedOrder, 'trades', []);
+        if (rawType === 'match') { // order is executed
+            trades.push (this.safeTrade ({
```

Comment:
@pcriadoperez I'm not sure we should do this, last time I checked `safeOrder` expected raw trades